sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/AppendBlockFromUrl.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.stress;
+
+import com.azure.core.util.Context;
+import com.azure.storage.blob.BlobAsyncClient;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.models.BlobRange;
+import com.azure.storage.blob.sas.BlobContainerSasPermission;
+import com.azure.storage.blob.sas.BlobServiceSasSignatureValues;
+import com.azure.storage.blob.specialized.AppendBlobAsyncClient;
+import com.azure.storage.blob.specialized.AppendBlobClient;
+import com.azure.storage.blob.stress.utils.OriginalContent;
+import com.azure.storage.stress.CrcOutputStream;
+import com.azure.storage.stress.StorageStressOptions;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.time.OffsetDateTime;
+
+public class AppendBlockFromUrl extends BlobScenarioBase<StorageStressOptions> {
```

Comment:
Does AppendBlockFromUrl need a stress test given this is a call for the service to handle the payload and I'm not sure too much is gained